src/chi/components/picker/picker.scss

```diff
@@ -0,0 +1,104 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+  .a-picker {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    input {
+      &[type='checkbox'],
+      &[type='radio'] {
+        &.a-input {
+          cursor: pointer;
+          left: -9999px;
+          position: absolute;
+
+          & + label {
+            background-color: $text-white;
+            border: solid 0.125rem $text-white;
+            border-radius: 5rem;
+            color: $dark-color;
+            font-size: $text;
+            height: 5rem;
+            justify-content: center;
+            line-height: 1rem;
+            margin: 0.5rem;
+            max-width: 8rem;
+            padding: 1.125rem;
+            transition: all .2s;
+            width: 8rem;
+
+            &:active {
+              background-color: set-color(grey, 20);
+            }
+
+            &:hover {
+              background-color: set-color(grey, 10);
+            }
+
+            &::after,
+            &::before {
+              display: none;
+            }
+          }
+
+          &:checked {
+            & + label {
+              border: solid 0.125rem $brand-color;
+            }
+          }
+
+          &.-withIcon {
```

Comment:
@davidojedacl let's rename to `-hasIcon` to be consistent with our other components.